### DISCUSSION TITLE
having option to reset interval

### DISCUSSION THREAD

**Discussion by rawand-faraidun** [mantinedev/mantine#5543] [Feature requests]
I think it would be awesome to have an option to reset the interval.

in my case, I have 2 tabs that are constantly switched every 5 seconds, but if the user switches the tabs themselves, it would be great to have a function like `interval.reset()`, It can be achieved it by using stop and start, but i still like the cleaner way
```
interval.stop()
interval.start()
```

**Comment by ryuhangyeong** [comment]
https://mantine.dev/hooks/use-interval/

is this what you want?

**Reply by rawand-faraidun** [reply]
I have used the interval but to reset the interval I used `stop` and `start` functions as mentioned in the discussion, but it would be awesome to have a function to reset with one line like `interval.restart` or something like that.
it wont affect that much, but something better to have
